### PR TITLE
actually use kots/kotsadm changes to test

### PR DESCRIPTION
**Pull request by salahalsaleh**

### CODE REVIEW

File: .github/workflows/build-test.yaml [hunk 1]
```diff
@@ -53,7 +59,15 @@ jobs:
...
-        run: ./bin/kots --kubeconfig=$GITHUB_WORKSPACE/.kubeconfig install qakots/github-actions --port-forward=false -n automated-$GITHUB_RUN_ID --shared-password password
+        run: |
+          ./bin/kots --kubeconfig=$GITHUB_WORKSPACE/.kubeconfig \
+          install qakots/github-actions \
+          --port-forward=false \
+          -n automated-$GITHUB_RUN_ID \
+          --shared-password password \
+          --kotsadm-registry ttl.sh \
+          --kotsadm-namespace automated-$GITHUB_RUN_ID \
+          --kotsadm-tag 2h
```
Comment (emosbaugh) [inline .github/workflows/build-test.yaml:70]: this is not unique. wont prs step on eachothers toes?

Comment (emosbaugh) [reply .github/workflows/build-test.yaml:70]: wait. i dont understand how UUID comes into play here

Comment (salahalsaleh) [reply .github/workflows/build-test.yaml:70]: `--kotsadm-namespace` is

Comment (salahalsaleh) [reply .github/workflows/build-test.yaml:70]: yeah, UUID should solve that

Comment (salahalsaleh) [reply .github/workflows/build-test.yaml:70]: --kotsadm-namespace takes care of that

**Pull request merged**